rollup.config.js

```diff
@@ -67,13 +68,38 @@ const external = (id) =>
 // Pragmatically create a Rollup config for each package
 const getRollupConfig = ({ pwd, babelConfigFile }) => {
   const SOURCE_DIR = path.resolve(pwd);
+
+  const HAS_CSS_ASSETS =
+    walkSync(SOURCE_DIR, {
+      globs: ['**/*.scss', '**/*.css'],
+    }).length > 0;
+
+  const HAS_PRIVATE_EXPORTS =
+    walkSync(SOURCE_DIR, {
+      globs: ['private.js'],
+    }).length > 0;
+
   // Get the package.json file
   const pkgConfig = require(`${SOURCE_DIR}/package.json`);
+
   // Relative input location for Rollup to bundle from
-  const input = [`${SOURCE_DIR}/src/index.js`, `${SOURCE_DIR}/src/private.js`];
+  const input = [`${SOURCE_DIR}/src/index.js`];
```

Comment:
It looks like there are some pretty big changes in this file... but I am pretty sure the cause of our rollup config not working on everything else is this line here where we have the private entry.

I am pretty sure Rollup will just not use any of the other configured plugins if it does not encounter any files of those types. I suspect we might just be able to update this one thing and have a conditional input for the private exports in Cascara.

Were we seeing errors about the CSS plugins not being used? We could likely just check for private exports and call it a day.

Reply:
We saw some errors regarding rollup css plugins in `@espressive/prop-types` package. That's why we changed it not to use those that plugin if it's not needed.

Reply:
ok, I tried reverting back to what it was and I didn't see the errors anymore, we are keeping all plugins.